### ISSUE TITLE
[Feature Request] Move API Keys to params

### ISSUE THREAD

**Issue by yuripourre** [joaomdmoura/crewAI#451]
When the API key is loaded from environment, it removes the ability to run multiple crews in parallel, using different API Keys.

The langchain tools work like this. Would be really nice to have that on CrewAI tools too.

**Comment by yuripourre** [comment]
Up
